### DISCUSSION TITLE
Is it possible to show tile bounds/coordinates easily on a TileLayer/MvtTileLayer for debugging.

### DISCUSSION THREAD

**Discussion by smcallis** [visgl/deck.gl#9072] [Q&A]
Sometimes I want to fetch a single tile for debugging and its hard to figure out which one I'm looking at.

**Comment by ibgreen** [comment]
- The following example might be useful as a reference: https://loaders.gl/examples/tiles/pmtiles
- It does this by hooking into the tile layer rendering, the relevant code is here: https://github.com/visgl/loaders.gl/blob/master/examples/website/tiles/components/tile-source-layer.ts#L285
